tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./node_modules/@jupiterone/integration-sdk-dev-tools/config/typescript",
   "compilerOptions": {
     "outDir": "dist",
-    "downlevelIteration": true
+    "skipLibCheck": true
```

Comment:
Do we want to add `"@types/node": "^14"` to our devDependencies instead of doing this?